packages/platform-browser/src/IndexedDbVersion.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * @since 1.0.0
+ */
+import * as Effect from "effect/Effect"
+import * as HashMap from "effect/HashMap"
+import type { Pipeable } from "effect/Pipeable"
+import { pipeArguments } from "effect/Pipeable"
+import * as IndexedDbDatabase from "./IndexedDbDatabase.js"
+import type * as IndexedDbQuery from "./IndexedDbQuery.js"
+import type * as IndexedDbTable from "./IndexedDbTable.js"
+import * as internal from "./internal/indexedDbQuery.js"
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export const TypeId: unique symbol = Symbol.for(
+  "@effect/platform-browser/IndexedDbVersion"
+)
+
+/**
+ * @since 1.0.0
+ * @category type ids
+ */
+export type TypeId = typeof TypeId
+
+/**
+ * @since 1.0.0
+ * @category interface
+ */
+export interface IndexedDbVersion<
+  out Tables extends IndexedDbTable.IndexedDbTable.AnyWithProps = never
+> extends Pipeable {
+  new(_: never): {}
+
+  readonly [TypeId]: TypeId
+  readonly tables: HashMap.HashMap<string, Tables>
+  readonly api: Effect.Effect<
```

Comment:
I think `getQueryBuilder` is a more obvious name, and the `IndexedDbQuery` module should be called `IndexedDbQueryBuilder`.

And a `IndexedDbQueryBuilder` returns a `IndexedDbQuery`.

Reply:
Renamed